torch/optim/rprop.py

```diff
@@ -305,8 +305,8 @@ def _view_complex_as_real(tensor_list):
 
         # update stepsizes with step size updates
         torch._foreach_mul_(grouped_step_sizes, signs)
-        for step_size in grouped_step_sizes:
-            step_size.clamp_(step_size_min, step_size_max)
+        torch._foreach_clamp_min_(grouped_step_sizes, step_size_min)
```

Comment:
This is quite unfortunate speed-wise as we do two passes over the data instead of one.
Would it be simple to add a foreach clamp?

Reply:
Will do simplifying code later to make rprop faster..am actually going to make an issue about RProp cuz there are several issues with it